OpenRA.Mods.Common/Pathfinder/CellInfoLayerPool.cs

```diff
@@ -15,7 +15,7 @@
 
 namespace OpenRA.Mods.Common.Pathfinder
 {
-	sealed class CellInfoLayerPool
+	public class CellInfoLayerPool
```

Comment:
think it best to keep it sealed.

Reply:
Not possible as Graph is public that is used by a PathSearch which is also public.

Reply:
`public sealed`